package.json

```diff
@@ -126,8 +126,8 @@
     "eslint-config-airbnb-base": "15.0.0",
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-chai-friendly": "0.7.2",
-    "eslint-plugin-import": "2.26.0",
-    "eslint-plugin-mocha": "10.0.4",
+    "eslint-plugin-import": "2.25.4",
+    "eslint-plugin-mocha": "10.0.3",
```

Comment:
Revert changes

Reply:
Reverted,  but  this prevents mocha from running:

ko5tik@roadrunner:~/sources/generator-jhipster$ npm run test

> generator-jhipster@7.8.1 pretest
> npm run lint


> generator-jhipster@7.8.1 lint
> npm run eslint && npm run ejs-lint


> generator-jhipster@7.8.1 eslint
> eslint . --ext .js,.cjs,.mjs


Oops! Something went wrong! :(

ESLint: 8.16.0

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Failed to load plugin 'mocha' declared in '.eslintrc.json': Package subpath './src/find' is not defined by "exports" in /home/ko5tik/sources/generator-jhipster/node_modules/ramda/package.json
    at new NodeError (node:internal/errors:377:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:440:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:719:3)
    at resolveExports (node:internal/modules/cjs/loader:483:36)
    at Module._findPath (node:internal/modules/cjs/loader:523:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:925:27)
    at Module._load (node:internal/modules/cjs/loader:780:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (/home/ko5tik/sources/generator-jhipster/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/home/ko5tik/sources/generator-jhipster/node_modules/eslint-plugin-mocha/lib/rules/handle-done-callback.js:3:14)